sources/packages/web/src/services/AuthService.ts

```diff
@@ -258,6 +266,13 @@ export class AuthService {
         await this.executeSiteminderLogoff(redirectUri);
         break;
       }
+      case ClientIdType.Student: {
+        if (options?.invalidBetaUser) {
+          redirectUri += "/login/invalid-beta-user";
+        }
+        await this.executeSiteminderLogoff(redirectUri);
```

Comment:
`executeSiteminderLogoff` is only applicable for IDIR and BCeID. It should not be invoked for BCSC.
We need to check if the student is authenticated using BCeID before calling `executeSiteminderLogoff`.